Makefile

```diff
@@ -105,10 +106,8 @@ help:
 	@echo " * 'shell' - Run the built image and attach to a shell"
 	@echo " * 'clean' - Clean artifacts"
 
-# Build a container image (skopeobuild) that has everything we need to build.
-# Then do the build and the output (skopeo) should appear in current dir
+# Do the build and the output (skopeo) should appear in current dir
 binary: cmd/skopeo
-	${CONTAINER_RUNTIME} build ${BUILD_ARGS} -f Dockerfile.build -t skopeobuildimage .
```

Comment:
Note the difference between `Dockerfile.build` and `Dockerfile`; this would break the following line.

(I don’t think any of our build processes use this target, so it might not be worth spending much time on changing it. OTOH some users do use this, at least from time to time, it seems…)

Reply:
I ran it locally and it worked.  All the packages from `Dockerfile.build` wound up in the image I built (mostly by accident).  I've made that explicit now - [Ref. packages.txt file in my other PR.](https://github.com/containers/automation_images/pull/83/files)

Reply:
I’m not saying we need two different containers (I haven’t checked); my point really is that this line used to build `skopeobuildimage` and the following line continues to run `skopeobuildimage`. So, [with the code the way I see it in this PR, which might not be relevant], it can only work if you have run the target previously and have a local image tagged `skopeobuildimage`, AFAICS.

Reply:
Oh...I thought I fixed that...

Reply:
...ya, it's fixed.  Everything now references `$(IMAGE)` which I need to figure out how to look up from `.cirrus.yml` to keep things consistent...

Reply:
...okay, this has been solved by re-using the get_ci_vm container image.  It contains a script that can properly resolve env. var. values from inside `.cirrus.yml`.